### ISSUE TITLE
Surveys in create posts button is not accessible

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4188]
**Describe the bug**
Surveys in create posts button is not accessible via keyboard.

**Where was the bug observed**
views/map in v5 deployment

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to '/view/maps'
2. Click on the '+' icon to create a new post.
3. See that the surveys are not accessible

**Screenshots**
![image](https://user-images.githubusercontent.com/70752431/102888904-a97cdd00-447f-11eb-83af-ebb6546dd315.png)
